Style Guide and Diataxis Review: Streams I (#227)
iotaledger/streams: documentation/docs/welcome.md

- library
- rust
- nodejs
- reference
---
# Welcome

> This section contains the documentation for the official IOTA Streams software. You can read more about the core principles behind IOTA Streams in our [streams blog post](https://blog.iota.org/iota-streams-alpha-7e91ee326ac0/).

Reviewer: ```suggestion
Welcome to the official documentation for the IOTA Streams software. You can read more about the core principles behind IOTA Streams in our [streams blog post](https://blog.iota.org/iota-streams-alpha-7e91ee326ac0/).
```
